src/cpu/jit_utils/jit_utils.cpp

```diff
@@ -18,15 +18,8 @@
 
 #include "utils.hpp"
 
-#ifndef DNNL_ENABLE_JIT_PROFILING
-#define DNNL_ENABLE_JIT_PROFILING 1
-#endif
-
-#ifndef DNNL_ENABLE_JIT_DUMP
-#define DNNL_ENABLE_JIT_DUMP 1
-#endif
-
 #if DNNL_ENABLE_JIT_PROFILING
+#define DNNL_ENABLE_JIT_DUMP 1
```

Comment:
The `DNNL_ENABLE_JIT_DUMP` flag is actually independent from the `DNNL_ENABLE_JIT_PROFILING` flag. Can you please keep it this way?

Reply:
I've put this back as it was and set DNNL_ENABLE_JIT_DUMP with DNNL_ENABLE_JIT_PROFILING in cmake/platform.cmake